app/_components/_admin/admin-blog-form.tsx

```diff
@@ -58,10 +64,14 @@ export default function AdminBlogForm({ id, title, tag, content, cover, draft =
           <p className="text-sm text-muted-foreground">{title}</p>
         </div>
         <div className="flex items-start gap-4">
-          <div className="grid w-full items-center gap-1.5">
-            <Input id="cover" type="file" name="cover" />
+          <div className="grid w-full items-center gap-1.5 relative">
+            <Input ref={inputRef} id="cover" type="file" name="cover" />
             <p className="text-sm text-muted-foreground">{cover ?? 'No cover provided'}</p>
+            <div className="absolute top-0 right-0 w-max items-center gap-1.5">
+              <Button variant="ghost" onClick={() => clearImage()} ><PlusIcon className='text-foreground size-5 rotate-45'  /></Button>
```

Comment:
as for `Button`, use `size` attr to `icon`,
like: `<Button size="icon"/>`
and adjust width and height with className if necessary